pagopa/io-app
# fix: [IA-257] Incorrect layout in several screens

## Short description
Several screens in the app don't respect the safe area of devices using the home indicator. This PR addresses it in all those screens (fixes #3364).

## List of changes proposed in this pull request
- A `SafeAreaView` has been added as a top level container to the following screens:

| Before  | After |
| ------------- | ------------- |
|![Simulator Screen Shot - iPhone 12 - 2022-02-04 at 18 50 04](https://user-images.githubusercontent.com/467098/152768201-34e0fbf3-64f4-4390-a65b-fa04ec1ca9f2.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-07 at 10 27 57](https://user-images.githubusercontent.com/467098/152768243-b8c1eda8-34c2-41f2-86a3-151898aa785d.png)|
|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 10 32 39](https://user-images.githubusercontent.com/467098/152768395-ce1884b0-d959-47e6-b7e4-6730ea70ec9d.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-04 at 15 26 14](https://user-images.githubusercontent.com/467098/152768418-6fd0cb57-a39d-4afa-a4d7-33f3b60a948e.png)|
|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 18 00 03](https://user-images.githubusercontent.com/467098/152768443-f6cd529d-bdce-4b4e-bb58-b768f7cd95aa.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 18 04 48](https://user-images.githubusercontent.com/467098/152768479-43d0ae77-0220-4efd-ae1d-9ac6678fd347.png)|
|![Simulator Screen Shot - iPhone 12 - 2022-02-04 at 18 48 54](https://user-images.githubusercontent.com/467098/152768612-91c4a205-b8b3-4a3a-b068-dd0992b5409a.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-07 at 11 15 56](https://user-images.githubusercontent.com/467098/152768818-c5e0121b-9d37-494a-b309-8ec46c530c8c.png)|
|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 17 16 42](https://user-images.githubusercontent.com/467098/152768920-52d308cd-b326-4fea-98dc-b093c2a3cbd3.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 17 16 19](https://user-images.githubusercontent.com/467098/152768978-bc8cedb8-00ee-49d9-96b7-bb8034ce5792.png)|
|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 15 50 50](https://user-images.githubusercontent.com/467098/152769043-9e048921-afdc-424e-90ef-9c9bfaa7df14.png)|![Simulator Screen Shot - iPhone 12 - 2022-02-03 at 15 50 34](https://user-images.githubusercontent.com/467098/152769073-d8731097-c0bd-479b-b39a-c7d97aa8600a.png)|

Although GitHub reports a lot of line changes, this is just because the indentation of the components of the screen have changed.

## How to test
Navigate throughout the screens and verify that there are no regressions. If you have a particular Android device that might be impacted by this PR, please let me know (I tested on iPhone 12, iPhone 8 and Pixel 3a).
